pkg/kv/postgres/store.go

```diff
@@ -62,16 +62,19 @@ func normalizeDBParams(p *kvparams.Postgres) {
 	}
 }
 
-func (d *Driver) Open(ctx context.Context, kvparams kvparams.KV) (kv.Store, error) {
+func (d *Driver) Open(ctx context.Context, kvParams kvparams.KV) (kv.Store, error) {
 	// TODO(barak): should we handle Open reuse the same store based on name
-	normalizeDBParams(kvparams.Postgres)
-	config, err := pgxpool.ParseConfig(kvparams.Postgres.ConnectionString)
+	if kvParams.Postgres == nil {
+		return nil, kv.ErrDriverConfiguration
+	}
```

Comment:
1. missing the same for dynamodb
2.
```suggestion
	if kvparams.Postgres == nil {
		return nil, fmt.Errorf("%w: missing %s settings", kv.ErrDriverConfiguration, DriverName)
	}

```